agentPresence/agentPresenceController.js

```diff
@@ -0,0 +1,87 @@
+var micc = new Micc('localhost');
+
+function login() {
+    micc = new Micc($('#miccServer').val());
+    micc.login(
+        $('#username').val(),
+        $('#password').val(),
+        connectToEmployeeHub
+    );
+}
+
+function connectToEmployeeHub(data) {
+    micc.getEmployeeStats('me', populateEmployeeInfo);
+    micc.getEmployeeBusyReasonCodes('me', populateBusyReasonCodes);
+
+    var miccSignalr = new MiccSignalR(data.miccServer, data.access_token);
+    miccSignalr.start(function () {
+        var hub = miccSignalr.createEmployeeHubProxy();
```

Comment:
What would be nice here, instead of a string slinging, would be a strongly typed wrapper.  Here's what I mean.   Instead of
```
var hub = miccSignalr.createEmployeeHubProxy();
miccSignalr.invokeHubMethod(hub, 'addSelfMonitor);
miccSignalr.onClientMethod(hub, 'employeeStateChanged', callback);
```
do something like
```
miccSignalr.addSelfMonitor();
miccSignalr.employeeStateChanged(callback);
```
* Bury the hub use and creation in the signalR class.
* Wrap string method names with a real function.  This aligns with all the route method hiding we did in the micc.js class.